docs/MapColonies/vector/Services/geocoding/README.md

```diff
@@ -0,0 +1,1024 @@
+            "results_count": 1,
+            "max_score": 2.0481892,
+            "match_latency_ms": 77
+        }
+    },
+    "features": [
+        {
+            "type": "Feature",
+            "geometry": {
+                "coordinates": [ [ [ 12.539507865186607, 41.851751203650096 ],
+                        [ 12.536787075186538, 41.94185043165008 ],
+                        [ 12.42879133518656, 41.93952837265009 ],
+                        [ 12.431625055186686, 41.84943698365008 ],
+                        [ 12.539507865186607, 41.851751203650096] ] ],
+                "type": "Polygon"
+            },
+            "properties": {
+                "TILE_NAME": "RIT",
+                "LAYER_NAME": "CONTROL.TILES",
+                "TYPE": "TILE",
+                "matches": [ { "layer": "CONTROL.TILES", "source": "control_gil_v5_test", "source_id": [] } ],
+                "names": { "default": [ "RIT" ], "display": "Tile RIT" },
+                "score": 2.0481892
+            }
+        }
+    ]
+}
+```
+</details>
+
+<details>
+    <summary>JSON Schema of Sub Tile Response</summary>
+
+```json
+{
+    "type": "FeatureCollection",
+    "geocoding": {
+        "version": "0.1.0",
+        "query": {
+            "tile": "RIT",
+            "sub_tile": "65",
+            "disable_fuzziness": true,
+            "limit": 5
+        },
+        "response": {
+            "results_count": 1,
+            "max_score": 3.588634,
+            "match_latency_ms": 5
+        }
+    },
+    "features": [
+        {
+            "type": "Feature",
+            "geometry": {
+                "coordinates": [ [ [ 12.439530324602458, 41.93031190061167 ],
+                        [12.439646505602468, 41.9393328566117 ],
+                        [ 12.429158378602494, 41.939432381611695 ],
+                        [ 12.429043189602453, 41.930411390611695 ],
+                        [ 12.439530324602458, 41.93031190061167 ] ] ],
+                "type": "Polygon"
+            },
+            "properties": {
+                "SUB_TILE_ID": "65",
+                "TILE_NAME": "RIT",
+                "LAYER_NAME": "CONTROL.SUB_TILES",
+                "TYPE": "SUB_TILE",
+                "matches": [ { "layer": "CONTROL.SUB_TILES", "source": "control_gil_v5_test", "source_id": [] } ],
+                "names": { "default": [ "65" ], "display": "Tile RIT Sub Tile 65" },
+                "score": 3.588634
+            }
+        }
+    ]
+}
+```
+</details>
+
+### Understanding the Control language
+Routes are roads and streams. Each Route has Control Points. Control Points are points scattered on the route. They are usefull when end-user reports it's location on the route. Its used to "pin-point" end-user location.<br/>
+Route name is just a string (in any length). For main roads, the route's name will be unique. For minor roads the name might be reused in other areas of interest. <br/>
+Route's Control point is a 3-digit number.
+
+<details>
```

Comment:
Same here, when giving a response example please elaborate on what the user is about to see and the context.